tests/infra_test.go

```diff
@@ -731,50 +731,33 @@ var _ = Describe("Infrastructure", func() {
 			}
 		})
 
-		It("[test_id:4143]should include the network metrics for a running VM", func() {
+		table.DescribeTable("should include metrics for a running VM", func(metricSubstring string) {
 			for _, url := range metricsURLs {
-				metrics := collectMetrics(url, "kubevirt_vmi_network_")
+				metrics := collectMetrics(url, metricSubstring)
 				By("Checking the collected metrics")
 				keys := getKeysFromMetrics(metrics)
 				for _, key := range keys {
 					value := metrics[key]
 					Expect(value).To(BeNumerically(">=", float64(0.0)))
 				}
 			}
-		})
-
-		It("[test_id:4144]should include the memory metrics for a running VM", func() {
-			for _, url := range metricsURLs {
-				metrics := collectMetrics(url, "kubevirt_vmi_memory")
-				By("Checking the collected metrics")
-				keys := getKeysFromMetrics(metrics)
-				for _, key := range keys {
-					value := metrics[key]
-					// swap metrics may (and should) be actually zero
-					Expect(value).To(BeNumerically(">=", float64(0.0)))
-				}
-			}
-		})
-
-		It("[test_id:4553]should include the vcpu wait metrics for running VM", func() {
-			for _, url := range metricsURLs {
-				metrics := collectMetrics(url, "kubevirt_vmi_vcpu_wait")
-				for _, v := range metrics {
-					fmt.Fprintf(GinkgoWriter, "vcpu wait was %f", v)
```

Comment:
Tihd print was omitted in the from the generic method.

Reply:
because the table entry print has enough info
see please 
https://github.com/kubevirt/kubevirt/pull/4140/files/27af4be91d1d719126f6d7f94282b9243b064efb..d604ea8a342c91b63ddc960dde9d56f9b1b460dd#diff-29539fc2ce2beadde4ecbf90b4a00816R749

Reply:
the `By` had enough info, but since i merged the tables, left the print value, done